test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -451,6 +429,126 @@ func TestMultiplePolicyBindings(t *testing.T) {
 	checkFailureReason(t, err, metav1.StatusReasonInvalid)
 }
 
+func TestCRDParams(t *testing.T) {
+	testcases := []struct {
+		name          string
+		crdName       string
+		crdKind       string
+		schema        string
+		crName        string
+		crSpec        map[string]interface{}
+		policy        *admissionregistrationv1alpha1.ValidatingAdmissionPolicy
+		policyBinding *admissionregistrationv1alpha1.ValidatingAdmissionPolicyBinding
+		namespace     *v1.Namespace
+		err           string
+		failureReason metav1.StatusReason
+	}{
+		{
+			name:    "a rule that uses data from a CRD param resource does NOT pass",
+			crdName: "config-resource",
+			crdKind: "ConfigResource",
+			crName:  "config-obj",
+			schema: `{
```

Comment:
Do we anticipate the CRD kind, name and schema to change across the test cases? If not we can probably remove them from the table and just set the values in the test below

Reply:
I think it'd probably be worth making the schema case-specific (that way we can test a wider variety of expressions, etc.), but I think we could keep the kind/name the same since I don't think that changes much in the way of coverage.

Reply:
(marking as resolved since the relevant constants have been removed)